src/types/redis_geo.cc

```diff
@@ -115,6 +115,11 @@ rocksdb::Status Geo::SearchStore(const Slice &user_key, GeoShape geo_shape, Orig
   if (point_type == kMember) {
     GeoPoint geo_point;
     auto s = Get(user_key, member, &geo_point);
+    // store key is not empty, try to remove it before returning.
+    if (!s.ok() && s.IsNotFound() && !store_key.empty()) {
+      auto del_s = ZSet::Del(store_key);
+      if (!del_s.ok()) return s;
```

Comment:
Should we return `del_s` here?

Reply:
right, originally i wrote s, then changed it to del_s but forgot to modify the latter part. thanks